nemo_skills/code_execution/math_grader.py

```diff
@@ -84,9 +84,33 @@
 import contextlib
 import re
 import signal
+from importlib.metadata import PackageNotFoundError, version
 from math import isclose
 from typing import Union
 
+from rich import print as rprint
```

Comment:
Let's not use rich as it's not available inside the sandbox where we actually run this code. While we might be able to modify our own setup to include it, we don't always have control over the sandbox environment if using cloud solutions e.g.

Reply:
That's a good point.